bower.json

```diff
@@ -57,7 +57,8 @@
     "DOMPurify": "0.8.3",
     "jquery-tiny-pubsub": "https://github.com/cowboy/jquery-tiny-pubsub.git#~0.7.0",
     "markdown-it": "7.0.1",
-    "bootstrap-switch": "3.3.2"
+    "bootstrap-switch": "3.3.2",
+    "At.js": "~1.5.2"
```

Comment:
This is okay. The issue was that Caret.js and jquery.caret conflict when they're used on the same page...but I ended up deciding to remove jquery.caret altogether in https://github.com/dimagi/commcare-hq/pull/13807